sllurp/llrp.py

```diff
@@ -1413,6 +1427,8 @@ def __init__(self, config_dict=None):
         self.impinj_tag_content_selector = None
         self.impinj_event_selector = None
 
+        self.llrp_keepalive_interval_ms = 60000  # in ms, 0 = no keepalive request sent to reader
```

Comment:
Everything here is llrp config related, so I think that you don't need to have the "llrp_" prefix for your variable.